public/app/themes/clarity/src/components/c-article-item/view-blog-feed.php

```diff
@@ -67,7 +69,7 @@
             ?>
 
         <div class="c-article-excerpt">
-            <p><?php echo $post['excerpt']['rendered']; ?></p>
+            <p><?php echo $post->post_excerpt; ?></p>
```

Comment:
`$post['excerpt']['rendered'];`

I think 'rendered' is a modified value that has been sanitised - we need to trace the include path back to understand if this is the case though 😞 sorry about that.

As mentioned, we could use `get_the_excerpt()`